lib/src/input_method_engine.dart

```diff
@@ -192,7 +192,7 @@ class ImeSimulator {
       'method': 'TextInputClient.updateEditingStateWithDeltas',
     });
 
-    await ServicesBinding.instance.defaultBinaryMessenger.handlePlatformMessage(
+    await _tester.binding.defaultBinaryMessenger.handlePlatformMessage(
```

Comment:
Why was this changed?

Reply:
That method has been deprecated after `v3.9.0-19.0.pre` as mentioned [here](https://api.flutter.dev/flutter/services/BinaryMessenger/handlePlatformMessage.html).

Reply:
Sounds good. Thanks.